README.md

```diff
@@ -1,27 +1,27 @@
 # training-specific-template
 
-This repository is intended to serve as a template for creating a repository for an individual CCDL workshop.
-The repository structure and use of GitHub pages is intended to gather all material required to administer a workshop in one user-friendly place. 
+This repository is intended to serve as a template for creating a repository for an individual Data Lab workshop.
+The repository structure and use of GitHub pages is intended to gather all material required to administer a workshop in one user-friendly place.
 We use a template repository approach for maintainability.
 
 ## Creating a repository for an individual training workshop
 
-First, you will need to create a new repository. 
+First, you will need to create a new repository.
 
-### CCDL workshops
+### Data Lab workshops
 
 This repository can be used as a template by anyone with read permissions.
-Select the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) and create the repository under the organization (typically `AlexsLemonade`) or user you would like to administer the repository. 
+Select the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) and create the repository under the organization (typically `AlexsLemonade`) or user you would like to administer the repository.
 We recommend setting the new repository to public. (When you use GitHub pages on a private repo, those pages are publicly accessible anyway.)
 
 ### Workshops hosted by others
 
 If you are an external user, you can either [file an issue to request read access](https://github.com/AlexsLemonade/training-specific-template/issues/new?assignees=&labels=request&template=request-read-access-to-use-template-for-a-workshop.md&title=%5BRequest+read+access%5D+) or fork the repository.
-If you request read access and it is granted, follow [the instructions above](#ccdl-workshops).
+If you request read access and it is granted, follow [the instructions above](#data-lab-workshops).
 To fork the repository, [follow the GitHub instructions for forking a repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo).
 
 **There are currently some important limitations to using this repository as a template for externally-hosted workshop.**
-If you are not using the CCDL hosted RStudio Server, there are a number of instructions in this repository that are about distributing credentials and logging into that server.
+If you are not using the Data Lab hosted RStudio Server, there are a number of instructions in this repository that are about distributing credentials and logging into that server.
```

Comment:
I feel like there should be some kind of dash between "Data Lab" and "hosted" but I will leave that grammar and type of dash to you!